stable/spring-cloud-data-flow/README.md

```diff
@@ -8,7 +8,7 @@ Pipelines consist of [Spring Boot](http://projects.spring.io/spring-boot/) apps,
 This chart will provision a fully functional and fully featured Spring Cloud Data Flow installation
 that can deploy and manage data processing pipelines in the cluster that it is deployed to.
 
-MySQL is used as the data store for Spring Cloud Data Flow state and either RabbitMQ or Kafka is used as the messaging layer for streaming apps to communicate with one another.
+MySQL is used as the default data store for Spring Cloud Data Flow state and RabbitMQ is used as the default messaging layer for streaming apps to communicate with one another.
```

Comment:
please revert this change, there is no need to add default in both these places.